runtime/mount/python_lib/lplots/widgets/row.py

```diff
@@ -0,0 +1,26 @@
+from typing import Literal
+
+from . import _emit, _state
+
+
+class RowWidgetState(_emit.WidgetState[Literal["row"], None]):
+    items: list[str]
+
+
+def w_row(
+    *,
+    key: str | None = None,
+    items: set[object],
```

Comment:
have it take a list & enforce the uniqueness here. drop the subsequent duplicates

Reply:
actually i would create a type `ValidRowWidget = CheckboxWidget | RadioWidget | ... `, since not all widgets can be in a w_row. that should also give you proper typing so you don't have to assert for the _key?